src/aics-image-viewer/components/App/index.tsx

```diff
@@ -483,19 +468,13 @@ export default class App extends React.Component<AppProps, AppState> {
     if (viewerConfig.maskAlpha !== undefined) {
       alphaLevel = [viewerConfig.maskAlpha];
     }
+    return alphaLevel;
+  }
 
-    let imageMask = alphaSliderToImageValue(alphaLevel);
-    let imageBrightness = brightnessSliderToImageValue(
-      userSelections[BRIGHTNESS_SLIDER_LEVEL],
-      userSelections[PATH_TRACE]
-    );
-    let imageDensity = densitySliderToImageValue(userSelections[DENSITY_SLIDER_LEVEL], userSelections[PATH_TRACE]);
-    let imageValues = gammaSliderToImageValues(userSelections[LEVELS_SLIDER]);
-    // set alpha slider first time image is loaded to something that makes sense
-    this.setUserSelectionsInState({ [ALPHA_MASK_SLIDER_LEVEL]: alphaLevel });
-
-    // Here is where we officially hand the image to the volume-viewer
-
+  // set up the Volume into the Viewer using the current initial settings
+  private placeImageInViewer(aimg:Volume, newChannelSettings?):void {
```

Comment:
Add spaces after types, and looks like the same definitions could be applied to `initializeNewImage` as well if interested. (Type for `newChannelSettings` will certainly have to wait for #57 though, per our earlier conversation.)

Reply:
i thought this would have been autoformatted better. I'll fix the whitespace and add Volume as a type everywhere I can.